frontend/packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts

```diff
@@ -233,107 +207,6 @@ describe('Interacting with a `OwnNamespace` install mode Operator (Prometheus)',
     });
   });
 
-  it('displays YAML editor for creating a new `Alertmanager` instance', async () => {
```

Comment:
These tests re-run the same tests as above for a different resource. There's nothing different here, and they flake frequently.